client/app/layouts/_application.sass

```diff
@@ -1,3 +1,15 @@
+// Product overrides of Boostrap & Patternfly variables
```

Comment:
@AllenBW I can't take credit for it :) , this comment comes from @epwinchell PR #119 that I used as base for my PR.